pandas/core/reshape.py

```diff
@@ -62,7 +64,7 @@ class _Unstacker(object):
     unstacked : DataFrame
     """
 
-    def __init__(self, values, index, level=-1, value_columns=None):
+    def __init__(self, values, index, level_num, value_columns=None):
 
         self.is_categorical = None
```

Comment:
can you put a Parameters section here. Specifically what are the guarantees now on level_num. I believe this can only be an integer or a list of integers (level numbers).